modules/iam/iam_policy.json

```diff
@@ -91,6 +91,17 @@
             "Resource": [
                 "arn:${partition}:dynamodb:${region}:${account_id}:table/${dynamodb_table}"
             ]
+        },
+        {
+            "Action": [
```

Comment:
Since these actions are specific to the auth policies, which are specified by the user, let's expose a new variable that accepts a user-provided iam policy document. We can then use the `aws_iam_policy_document` with its `source_json` and `override_json` arguments to merge the user input with the policy required to manage the resources specific to this module.

Alternatively, we could accept a policy_arn, and let the user create their own policy and then attach it if provided.